test/forge/DeploymentTest.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import "./helpers/IntegrationTest.sol";
+
+contract DeploymentTest is IntegrationTest {
+    function testDeployMetaMorphoAddresssZero() public {
+        vm.expectRevert(ErrorsLib.ZeroAddress.selector);
+        new MetaMorpho(OWNER, address(0), ConstantsLib.MIN_TIMELOCK, address(loanToken), "MetaMorpho Vault", "MMV");
+    }
+
+    function testDeployMetaMorphoNotToken(address notToken) public {
+        vm.assume(address(notToken) != address(loanToken));
+        vm.assume(address(notToken) != address(collateralToken));
+
+        vm.expectRevert();
+        new MetaMorpho(OWNER, address(morpho), ConstantsLib.MIN_TIMELOCK, notToken, "MetaMorpho Vault", "MMV");
+    }
+
+    function testDeployMetaMorpho(
```

Comment:
This test is redundant with `MetaMorphoFactoryTest.testCreateMetaMorpho` (i would simply remove it)

Reply:
One is testing that `MetaMorphoFactory` is working as expected, the other is testing that deploying a MM works as expected. I think this is a little bit different no?

Reply:
Alright!